mesh/mesh.hpp

```diff
@@ -332,12 +332,11 @@ class Mesh
    /** Also, initializes #mesh_geoms. */
    void SetMeshGen();
 
-   /// Return the length of the segment from node i to node j.
-   double GetLength(int i, int j) const;
+   /// Computes the squared edge lengths for all edges in the mesh.
+   void GetEdgeLengths(const DSTable &v_to_v, Array<double> &lengths) const;
```

Comment:
There are several existing functions with `v_to_v` as an argument, which is never described. It would be helpful to mention in at least one place that it is constructed by `GetVertexToVertexTable`.

Reply:
See [6347978](https://github.com/mfem/mfem/pull/3847/commits/6347978eb2b59f1056317f86a1edc197ebf1e7bf).